content/2_Bronze/Intro_Greedy.mdx

```diff
@@ -179,3 +180,65 @@ programmers refer to this as "Proof by AC," or "Proof by Accepted."
 <Problems problems="general" />
 
 <!-- Anything else major that needs to be added? Bronze Ad Hoc is not very complicated. -->
+
+## Quiz
+	
+<Quiz>
+  <Quiz.Question>
+    What's a greedy algorithm?
+    <Quiz.Answer>
+      An algorithm which takes back choices in order to find the most optimal solution.
```

Comment:
i feel like this is too obvious. I think previously, the backtracking was good, as in the question. The answer would be something like "although greedy algorithms can use backtracking, this is not true for all of them"